src/disease/schemas.py

```diff
@@ -88,6 +88,36 @@ class NamespacePrefix(Enum):
     WIKIDATA = "wikidata"
 
 
+# Source to URI (from identifiers.org, if found)
+NAMESPACE_TO_SYSTEM_URI: dict[NamespacePrefix, str] = {
+    NamespacePrefix.NCIT: "https://www.ebi.ac.uk/ols4/ontologies/ncit/classes?short_form=NCIT_",
+    NamespacePrefix.MONDO: "https://monarchinitiative.org/MONDO:",
+    NamespacePrefix.DO: "https://www.ebi.ac.uk/ols4/ontologies/doid/terms?obo_id=DOID:",
+    NamespacePrefix.DOID: "https://www.ebi.ac.uk/ols4/ontologies/doid/terms?obo_id=DOID:",
+    NamespacePrefix.OMIM: "https://www.omim.org/entry/",
+    NamespacePrefix.ONCOTREE: "https://oncotree.mskcc.org/?version=oncotree_latest_stable&field=NAME&search=",
+    NamespacePrefix.EFO: "https://www.ebi.ac.uk/efo/EFO_",
+    NamespacePrefix.HP: "https://hpo.jax.org/app/browse/term/HP:",
+    NamespacePrefix.HPO: "https://hpo.jax.org/app/browse/term/HP:",
+    NamespacePrefix.KEGG: "https://www.kegg.jp/entry/",
+    NamespacePrefix.MEDDRA: "https://purl.bioontology.org/ontology/MEDDRA/",
+    NamespacePrefix.MEDGEN: "https://www.ncbi.nlm.nih.gov/medgen/",
+    NamespacePrefix.MESH: "https://id.nlm.nih.gov/mesh/",
+    NamespacePrefix.MP: "https://www.ebi.ac.uk/ols4/ontologies/mp/terms?obo_id=MP:",
+    NamespacePrefix.OBI: "https://purl.obolibrary.org/obo/",
+    NamespacePrefix.ORPHANET: "https://www.orpha.net/consor/cgi-bin/OC_Exp.php?Lng=EN&Expert=",
+    NamespacePrefix.PATO: "https://www.ebi.ac.uk/ols4/ontologies/pato/terms?obo_id=PATO:",
+    NamespacePrefix.UMLS: "https://linkedlifedata.com/resource/umls/id/",
+    NamespacePrefix.WIKIPEDIA: "https://en.wikipedia.org/wiki/",
+    NamespacePrefix.WIKIDATA: "https://www.wikidata.org/entity/",
```

Comment:
these would be my suggestions

* oncotree is pretty arbitrary but the below is what's provided by the bioregistry

```suggestion
    NamespacePrefix.NCIT: "http://purl.obolibrary.org/obo/NCIT_",
    NamespacePrefix.MONDO: "http://purl.obolibrary.org/obo/MONDO_",
    NamespacePrefix.DO: "http://purl.obolibrary.org/obo/DOID_",
    NamespacePrefix.DOID: "http://purl.obolibrary.org/obo/DOID_",
    NamespacePrefix.OMIM: "https://omim.org/MIM:",
    NamespacePrefix.ONCOTREE: "http://oncotree.mskcc.org/api/tumorTypes/search/code/",
    NamespacePrefix.EFO: "https://www.ebi.ac.uk/efo/EFO_",
    NamespacePrefix.HP: "http://purl.obolibrary.org/obo/HP_",
    NamespacePrefix.HPO: "http://purl.obolibrary.org/obo/HP_",
    NamespacePrefix.KEGG: "https://www.kegg.jp/entry/",
    NamespacePrefix.MEDDRA: "https://purl.bioontology.org/ontology/MEDDRA/",
    NamespacePrefix.MEDGEN: "https://www.ncbi.nlm.nih.gov/medgen/",
    NamespacePrefix.MESH: "https://id.nlm.nih.gov/mesh/",
    NamespacePrefix.MP: "http://purl.obolibrary.org/obo/MP_",
    NamespacePrefix.OBI: "http://purl.obolibrary.org/obo/OBI_",
    NamespacePrefix.ORPHANET: "https://www.orpha.net/consor/cgi-bin/OC_Exp.php?Lng=EN&Expert=",
    NamespacePrefix.PATO: "http://purl.obolibrary.org/obo/PATO_",
    NamespacePrefix.UMLS: "https://linkedlifedata.com/resource/umls/id/",
    NamespacePrefix.WIKIPEDIA: "https://en.wikipedia.org/wiki/",
    NamespacePrefix.WIKIDATA: "https://www.wikidata.org/entity/",
```

Reply:
I'm going to follow the description in [this](https://github.com/ga4gh/gks-core/pull/107) PR 

> The terminology/code system that defined the code. May be reported as a free-text name 
(e.g. 'Sequence Ontology'), but it is preferrable to provide an IRI or URL that resolves to 
          a specification of the system, or its home page. e.g. for the Sequence Ontology, 
          'http://purl.obolibrary.org/so.owl' resolves to an owl rendering of the ontology, and
          'http://www.sequenceontology.org/' resolves to its home page.